src/show.jl

```diff
@@ -1,4 +1,5 @@
 Base.show(io::IO, j::Object) = _show(io, j)
+Base.show(io::IO, j::Array) = _show(io, j)
```

Comment:
This was taken out recently because people wanted the normal fallback `AbstractArray` show method

Reply:
Ah sorry let me take that out again, I assumed it was an oversight.